frontend_lib/src/helper.js

```diff
@@ -112,47 +112,58 @@ export const BREADCRUMBS_TYPE = {
 export const revisionTypeList = [{
   id: 'archiving',
   faIcon: 'fas fa-archive',
-  label: i18n.t('Item archived')
+  tradKey: i18n.t('Item archived'),
+  label: 'Item archived'
 }, {
   id: 'content-comment',
   faIcon: 'far fa-comment',
-  label: i18n.t('Comment')
+  tradKey: i18n.t('Comment'),
+  label: 'Comment'
 }, {
   id: 'creation',
   faIcon: 'fas fa-magic',
-  label: i18n.t('Item created')
+  tradKey: i18n.t('Item created'),
+  label: 'Item created'
 }, {
   id: 'deletion',
   faIcon: 'far fa-trash-alt',
-  label: i18n.t('Item deleted')
+  tradKey: i18n.t('Item deleted'),
+  label: 'Item deleted'
 }, {
   id: 'edition',
-  faIcon: 'fas fa-edit',
-  label: i18n.t('New revision')
+  faIcon: 'fas fa-history',
+  tradKey: i18n.t('New revision'),
+  label: 'New revision'
 }, {
   id: 'revision',
   faIcon: 'fas fa-history',
-  label: i18n.t('New revision')
+  tradKey: i18n.t('New revision'),
+  label: 'New revision'
 }, {
   id: 'status-update',
   faIcon: 'fas fa-random',
-  label: statusLabel => i18n.t('Status changed to {{status}}', { status: statusLabel })
+  tradKey: 'Status changed to {{status}}',
+  label: (statusLabel) => i18n.t('Status changed to {{status}}', { status: statusLabel })
```

Comment:
Shouldn't it be the other way around?
the i18n on the tradKey and not on the label

Reply:
ah, yes. I'm fixing this.

Reply:
actually, the tradKey is useless here, so I'm just removing this line.